### PR TITLE
Casmcms-7611: Changes to BOS charts with updated explicit images

### PR DESCRIPTION
**Pull request by jsl-hpe**
## Summary and Scope

Included are build changes required to comply with updated base charts that include accurate image references, as well as changes to the way BOS is packaged as a helm chart. This includes changes that allow for local testing of the BOS charts before pushing code to github for build tests.

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* CASMCMS-7611
* Changes will need to be promoted to master as well as release/1.2

## Testing

- Mug & Rocket (where available); HELM update tests using a manifest to consume charts from a local directory <path>.

### Tested on:

  * mug
  * rocket

### Test description:

loftsman update commands were used with a manifest file to test changes.

## Risks and Mitigations

The base charts and underlying build repositories are always changing day to day. What works now may not work later, but by no fault of this particular commit.

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Target branch correct
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by rkleinman-hpe — APPROVED** [review]

File: kubernetes/cray-bos/Chart.yaml.in
Comment (rbak-hpe) [inline kubernetes/cray-bos/Chart.yaml.in:27]: According to Zach these maintainer names should be our hpe github usernames.  e.g. rbak-hpe

**Review by jsollom-hpe — APPROVED** [review]
Approved assuming you make/made the same corrections here as you did for the master branch.

**Pull request merged**
